test/RevenuePool-test.js

```diff
@@ -681,6 +684,33 @@ contract("RevenuePool", (accounts) => {
       ).to.equal("did:cardstack:56d6fc54-d399-443b-8778-d7e4512d3a49-xx");
     });
 
+    it("allows the merchant registrar to directly add a merchant", async () => {
+      let merchantTx = await merchantManager.registerMerchant(
+        merchantThree,
```

Comment:
It's confusing to me that the test starts with `merchantThree`. This naming implies that there are merchants 1 and 2 in play here but they are not. However I think I see the problem - the test assertions are all sharing state set in the `before` hook. I think it would be better to change that to `beforeEach` so that every assertion gets its own fresh state, and you could simply use the `merchant` variable, but maybe out of scope for this task. Otherwise I suggest you start the test with: `let merchant = accounts[N]` like a couple of other asssertions do.